backend/backend/api/routes.py

```diff
@@ -346,4 +353,18 @@ async def github_comment(repo, pr_number, request):
         with gr.Column():
             github_comment_output = gr.Textbox(label="Response")
     github_comment_button.click(github_comment, inputs=[
```

Comment:
The inputs to the github_comment_button.click method have been changed. The request_input is no longer included in the list of inputs.